programmers/k-th-number.js

```diff
@@ -0,0 +1,15 @@
+function solution(arry, commands){
+    const cd_len = commands.length;
+    let results = [];
+    for (let i = 0 ;  i < cd_len ; i++){
+        let start = commands[i][0]-1;
+        let end = commands[i][1];
+        let k = commands[i][2]-1;
+        let arryCuted = (arry.slice(start,end)).sort();
```

Comment:
sort() 메소드는 기본적으로 문자열 유니포인터를 기준으로 정렬합니다.
[1, 21, 30, 10000].sort() -> [1, 100000, 21, 30]
그렇기 때문에 숫자 정렬에서는 sort()만 쓰시면 안됩니다.
```suggestion
        let arryCuted = (arry.slice(start,end)).sort((a, b) => a - b);
```